classes/01class/exercises/c01-git02/starkmatt/c01-git02-1.txt

```diff
@@ -0,0 +1,6 @@
+Is it possible to see the inner commit of squashed commit with 'git rebase' ?
+
+Yes, using the command git reflog.
+Basically, reflog is an alias for git log. 
```

Comment:
Reflog is a local log of Git commands executed, and it makes easy to undo any commands applied.

Git log will list the commit history.

They are different since reflog comes empty every time you clone the repo.

Can you update this?
Thanks

Reply:
Done!